test/transaction-runner.ts

```diff
@@ -401,9 +402,9 @@ describe('TransactionRunner', () => {
           fakeError.code = status.UNKNOWN;
 
           fakeTransaction.request.onCall(0).callsFake(
-              (_, callback) => callback(fakeError));
+              (_: object, callback: Function) => callback(fakeError));
```

Comment:
use {} instead of object type, follow the same!

Reply:
Done!